### ISSUE TITLE
Fix in optimization doc on website

### ISSUE THREAD

**Issue by JulesGM** [Lightning-AI/pytorch-lightning#14595]
https://pytorch-lightning.readthedocs.io/en/stable/common/optimization.html#learning-rate-scheduling
says to look at the documentation for `configure_optimizers` but doesn't give a link. It would be helpful to just link https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html#configure-optimizers imho

cc @borda @rohitgr7 @Felonious-Spellfire @akihironitta

**Comment by arcAman07** [comment]
I would love to work on this issue!

**Comment by akihironitta** [comment]
@arcAman07 Thank you! That would be fantastic! :)

**Comment by turian** [comment]
If I have multiple optimizers and schedulers as per [the doc](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html#configure-optimizers) but I'm not training a GAN, just a single model, what happens?

Can I make it that the first scheduler and optimizer runs, then when the first scheduler stops (e.g. reaches plateau or whatever) it switches to the second scheduler + optimizer? If so can that be documented and an example given?

**Comment by awaelchli** [comment]
Links are working again, this was fixed 👍
